### DISCUSSION TITLE
Nvim-Lsp-Installer report says that pip3 is not available, is it a bug?

### DISCUSSION THREAD

**Discussion by tony-sol** [williamboman/nvim-lsp-installer#843] [Q&A] [answered]
When i do `:checkhealth` command in nvim, Nvim-Lsp-Installer report says that pip3 is not installed, but it in common $PATH (pic.related).
Is this a bug or just missconfiguration?
<img width="1138" alt="Снимок экрана 2022-06-30 в 21 04 03" src="https://user-images.githubusercontent.com/33671815/176746753-e57c26ec-b0e2-425e-baba-2a99a9f82327.png">

Also, same about java and javac

**Comment by williamboman** [comment, marked as answer]
Hello! It checks the installed pip version via `python3 -m pip --version` (and not via a `pip3` executable directly). `java` and `javac` are checked by `java(c) -version`. Does running those commands manually work fine?

**Reply by tony-sol** [reply]
Hi! Thanks for your reply
I thought it checked by having that in $PATH by this way you said - seems fair:)
